sc/social/like/browser/images/ico-email.svg

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
from where did you get this file? that's not the one we have talked about, but this one:

https://upload.wikimedia.org/wikipedia/commons/d/df/Aiga_mail.svg